README.md

```diff
@@ -31,35 +33,43 @@ type Response = [{
 }]
 ```
 
+To get synonyms for an ontology term Metis uses Iris to register a `action.get.synonyms` pattern.
+The synonyms payload must be in the [format](schemas/synonyms-is-valid.json):
+```
+{ ontologyIRI: string, ontologyShortName: string }
+```
+
+
+>>>>>>> origin/master
 
 ## Setting up the database
 
 Metis uses postgres to store a lookup table and accelerate the process of returning existing results from previous queries. To setup the database create a new  database called `metis` and execute [`setup.sql`](setup.sql).
 
 ## About Docker Compose
 
-The current version of [Iris](https://github.com/repositive/iris-js) requires an AMQP server. The docker-compose comes preconfigured to connect to one out of the box. You'll need to run an instance yourself and attach it to the network rabbit, and to the postgres database:
+The current version of [Iris](https://github.com/repositive/iris-js) requires an AMQP server. The docker-compose comes preconfigured to connect to one out of the box. You'll need to run an instance yourself and attach it to the network rabbit, and to the postgres database. For this purpose please use the devops service:
 
-**Create the postgres network**
-```bash
-$ docker network create postgres
-```
+**Use devops service to start postgres network and rabbit network**
```

Comment:
Perhaps this should be included as an alternative to creating the rabbit and postgres networks manually?

Reply:
Done.